### ISSUE TITLE
self.genome_id and self.data['build']

### ISSUE THREAD

**Issue by dongbohu** [greenelab/cimr#62]
I think this line:
https://github.com/greenelab/cimr/blob/607730b914072bf07f3a347c1fa21b8b2eaae2b3/cimr/processor/process.py#L418
should be changed to:
```
self.data['build'] = 'b38'
```

As a matter of fact, when the data is processed in chunks, because `self.genome_build` was changed to `b38` in the original line, `call_liftover()` wouldn't be invoked in any chunks other than the first one. So the output files would be different when `chunksize` is modified.

**Comment by dongbohu** [comment]
I realized that this line
```
self.data['build'] = 'b38'
```
is not unnecessary in `process.py`, because it has been self.data['build'] has been reset here:
https://github.com/greenelab/cimr/blob/607730b914072bf07f3a347c1fa21b8b2eaae2b3/cimr/processor/lift.py#L87

So this line:
https://github.com/greenelab/cimr/blob/607730b914072bf07f3a347c1fa21b8b2eaae2b3/cimr/processor/process.py#L418
should be removed so that it won't affect data processing of any chunks after the first one. To make it better, `self.genome_build` should be set as a read-only attribute.

**Comment by dongbohu** [comment]
According to:
https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.copy.html
This line:
https://github.com/greenelab/cimr/blob/d61b328c4564d2ff138d67f56b55da37493bdb80/cimr/processor/process.py#L443
does a deep copy of `data` (a pandas DataFrame). Since `data` won't be used afterwards, a shallow copy or a simple re-assignment would be good enough:
```python
self.data = data
```
This will also save the memory usage.

**Comment by dongbohu** [comment]
Fixed by PR #64.
